keel-core/src/main/kotlin/com/netflix/spinnaker/keel/verification/VerificationRunner.kt

```diff
@@ -26,6 +26,28 @@ class VerificationRunner(
    * particular environment and artifact version.
    */
   fun runVerificationsFor(context: VerificationContext) {
+    if (context.environment.verifyWith.isEmpty()) {
+      return
+    }
+
```

Comment:
I am having a hard time absorbing what this chain of things is doing. Could you add a quick comment(in the filter not block, the let block, and/or the return statement in the let), or change some variable names away from `it`, to help clarify what you're doing? I feel like if I had to debug this statement I'd shake my fist at you :D

Reply:
I tidied it up a bit & added some comments

Reply:
This is super helpful, thank you!!